src/ucs/config/global_opts.c

```diff
@@ -152,10 +151,6 @@ static ucs_config_field_t ucs_global_opts_table[] = {
    "Enable memory type (cuda/rocm) cache",
    ucs_offsetof(ucs_global_opts_t, enable_memtype_cache), UCS_CONFIG_TYPE_TERNARY},
 
- {"ASYNC_MAX_EVENTS", "1024", /* TODO remove this; resize mpmc */
```

Comment:
instead of removing , mark it as UCS_CONFIG_TYPE_DEPRECATED

Reply:
done